frontend/src/components/maps/leaflet/InfoSlideOut/InfoContent.tsx

```diff
@@ -113,8 +117,23 @@ export const InfoContent: React.FC<IInfoContent> = ({
             leftSideLabel={'Classification'}
             rightSideItem={propertyInfo?.classification}
           />
+          {!!propertyInfo?.projectNumbers?.length && (
+            <ThreeColumnItem
+              leftSideLabel={
+                propertyInfo.projectNumbers.length > 1 ? 'Project Numbers' : 'Project Number'
+              }
+              rightSideItem={propertyInfo.projectNumbers.map((projectNum: string) => (
+                <ProjectNumberLink
+                  setPrivateProject={setPrivateProject}
```

Comment:
What is the private project part about?

Reply:
Was just to see if the project was outside of the agency so it can display the contract SRES block in the parent